src/pages/sign-in.tsx

```diff
@@ -57,7 +58,7 @@ const SignIn = () => {
 
 export const getServerSideProps = withSessionSsr(
```

Comment:
are we still using SSR here? I would have thought the intention was to migrate to CSR fully.

Reply:
Will look to do some experimentation with CSR for auth later. I forgot what the exact blocker was in the sea of blockers i was dealing with. I think it had something to do with infinite redirects but it should've been solved with my other refactors